scripts/test/run-tests.sh

```diff
@@ -21,15 +21,15 @@ BBB_DEBIAN_SDCARD_IMAGE_URL="https://debian.beagleboard.org/images/bone-debian-1
 # Not on official home page, but found via https://elinux.org/Beagleboard:BeagleBoneBlack_Debian:
 
 ## Auto-update
-BBB_DEBIAN_EMMC_IMAGE_URL="https://rcn-ee.com/rootfs/bb.org/testing/2021-07-26/buster-console/bone-debian-10.10-console-armhf-2021-07-26-1gb.img.xz"
+BBB_DEBIAN_EMMC_IMAGE_URL="https://rcn-ee.com/rootfs/bb.org/testing/2021-08-09/buster-console/bone-debian-10.10-console-armhf-2021-08-09-1gb.img.xz"
 
 ## Auto-update
 RASPBIAN_IMAGE_URL="http://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2020-02-14/2020-02-13-raspbian-buster-lite.zip"
 
 UBUNTU_IMAGE_URL="https://downloads.mender.io/mender-convert/images/Ubuntu-Bionic-x86-64.img.gz"
 
 ## Auto-update
-UBUNTU_SERVER_RPI_IMAGE_URL="http://cdimage.ubuntu.com/ubuntu/releases/20.04.2/release/ubuntu-20.04.2-preinstalled-server-armhf+raspi.img.xz"
+UBUNTU_SERVER_RPI_IMAGE_URL=http://cdimage.ubuntu.com/ubuntu/releases/21.04/release/ubuntu-21.04-preinstalled-server-armhf+raspi.img.xz
```

Comment:
@oleorhagen: Do you know if there is a way to make it ignore non-LTS upgrades? If nothing else we could make sure the first number is divisible by 2, and the second is "04", which corresponds to their schedule.

Reply:
I have this on my TODO list actually, and I have not come up with anything smarter than what you said above, but I guess it should suffice

Reply:
Follow-up: https://github.com/mendersoftware/mender-convert/pull/374